plugins/module_utils/dnac.py

```diff
@@ -158,6 +158,41 @@ def get_task_details(self, task_id):
 
         return result
 
+    def check_task_response_status(self, response):
+        """
+        Get the site id from the site name.
+
+        Parameters:
+            self - The current object details.
+            response (dict) - API response
+
+        Returns:
+            self
+        """
+
+        if not response:
+            self.msg = "response is empty"
+            self.status = "exited"
+            return self
+
+        if not isinstance(response, dict):
+            self.msg = "response is not a dictionary"
+            self.status = "exited"
+            return self
+
+        task_id = response.get("response").get("taskId")
+        while True:
+            task_details = self.get_task_details(task_id)
+            self.log(str(task_details))
+            if task_details.get("isError") is True:
```

Comment:
one quick question. task_details.get("isError").. It could be either True or false..  Then what is the need for this While loop. Are we missing something?